sandbox/tests/collision/touching.ts

```diff
@@ -4,7 +4,7 @@ var game = new ex.Engine({
     height: 200,
     width: 800
 });
-var paddle = new ex.Actor(150, game.getDrawHeight() - 40, 200, 20);
```

Comment:
I believe our policy is to deprecate old methods and then replace them after one milestone. Could you leave these in with `@deprecated` and then move the changelog entry to 'Deprecated'?